robert_beverage/scripts/web_server

```diff
@@ -27,6 +36,37 @@ def isfloat(value):
         return False
 
 
+def get_coordinates():
+    while True:
+        try:
+            trans = tfBuffer.lookup_transform('map', 'base_link', rospy.Time())
+        except (tf2_ros.LookupException, tf2_ros.ConnectivityException,
+                tf2_ros.ExtrapolationException):
+            rate.sleep()
+            continue
+        else:
+            break
+    return {'x': trans.transform.translation.x,
+            'y': trans.transform.translation.y}
+
+
+def move_base(x, y):
+    cur_coor = get_coordinates()
+    goal = MoveBaseGoal()
+    goal.target_pose.header.frame_id = "map"
+    goal.target_pose.header.stamp = rospy.Time.now()
+    goal.target_pose.pose.position.x = cur_coor['x'] - x
```

Comment:
your `x` and `y` coordinates are already provided in `map` frame. You do not need to convert it to any other frame. Just use here `x` and `y`. Also you do not need `cur_coor` at all.
If you made these changes everything should work.